docs/usage/misc/logging.md

```diff
@@ -29,3 +30,33 @@ rule 'foo' do
   on_start
 end
 ```
+
+## Logger Methods
+
+The `logger` object has the following methods
+
+| Method            |
+| ----------------- |
+| info              |
+| warn              |
+| error             |
+| debug             |
+| trace             |
+| is_info_enabled?  |
```

Comment:
Unrelated to the review, but we should fix those names in another PR.  Should be info?, warn?, etc.

Reply:
good catch!